app/controllers/items_controller.rb

```diff
@@ -0,0 +1,44 @@
+class ItemsController < ApplicationController
+  def index
+    if params[:merchant_id].nil?
```

Comment:
you can just say `if params[:merchant_id]` without the nil because if it is there it is still "truthy", if it's not there, it will be nil.